src/tests/users.test.ts

```diff
@@ -0,0 +1,207 @@
+import { expect, test } from '@/fixtures/fixtures';
+import Navbar from '@/pages/Navbar';
+import AppConfig from '@/utils/AppConfig';
+
+test.beforeEach(async ({     
+    page,
+ }) => {
+    await page.goto('./dashboard');
+  });
+
+//OBPIH-4622 Users
+test('Create user, add default location and auto-login and delete user', async ({navbar, userListPage, createUserPage, editUserPage, page, loginPage, locationChooser}) => {
+  test.setTimeout(3 * 60 * 1000);
+    await navbar.configurationButton.click();
+    await navbar.getNavItem('Users').click();
+    await userListPage.createUserButton.click();
+    await createUserPage.username.fill('test1');
+    await createUserPage.firstName.fill('test1');
+    await createUserPage.lastName.fill('test1');
+    await createUserPage.password.fill('test1');
+    await createUserPage.saveButton.click();
+    await expect(editUserPage.userTitle).toContainText('test1 test1');
+    await editUserPage.activateUser.click();
+    await editUserPage.saveButton.click();
+    await editUserPage.authorizationTab.click();
+    await editUserPage.defaultRole.click();
+    await editUserPage.getUserRole('Manager').click();
+    await editUserPage.defaultLocation.click();
+    await editUserPage.getDefaultLocation('main location').click();
```

Comment:
`'main location'` I assume it is a name of a location that is created in some environment like obdev3 but might be different in other locations.

If I understand correctly this is a name of a location `nomanageinventorydepot`, if so then you should use the `nomanageinventorydepot` from AppConfig that is.

I suggest creating a locationService fixture like eg.
```ts
  noManageInventoryDepot: async ({ page }, use) =>
    use(new LocationData('noManageInventoryDepot', page.request)),
```
and then fetch this location information and use the name or organization of this location in tests
```ts
const location = await noManageInventoryDepot.getLocation()
await editUserPage.getDefaultLocation(location.name).click();

```